src/main_menu.cpp

```diff
@@ -68,10 +68,20 @@ class demo_ui : public cataimgui::window
         void on_resized() override {
             init();
         };
+
+    private:
+        std::shared_ptr<cataimgui::Paragraph> stuff;
 };
 
 demo_ui::demo_ui() : cataimgui::window( _( "ImGui Demo Screen" ) )
 {
+    // NOLINTBEGIN(cata-text-style)
+    // char *text = "Some long text that will wrap around nicely. </color><color_green><color_red>Some red text in the </color>middle.</color> Some long text that will <color_light_blue_yellow>wrap around nicely.";
+    std::string text =
+        "Some long text that will wrap around nicely. <color_red>Some red text in the middle.</color> Some long text that will wrap around nicely.";
+    // NOLINTEND(cata-text-style)
```

Comment:
```suggestion
    // char *text = "Some long text that will wrap around nicely. </color><color_green><color_red>Some red text in the </color>middle.</color> Some long text that will <color_light_blue_yellow>wrap around nicely.";
    std::string text =
        "Some long text that will wrap around nicely.  <color_red>Some red text in the middle.</color>  Some long text that will wrap around nicely.";
```

Even if it's only demo code, just adding the extra spaces is preferable over ignoring the error.